defaults/main.yml

```diff
@@ -1,11 +1,32 @@
-# Dont change this value, it's value is referenced
+
+# Tarsnap key contents
+# Should look like
+# TARSNAP_KEY: |
+#      # START OF TARSNAP KEY FILE
+#      contents
+#      # END OF TARSNAP KEY FILE
+TARSNAP_KEY: null
 TARSNAP_KEY_REMOTE_LOCATION: /root/tarsnap.key
-TARSNAP_KEY_LOCAL_TEMPLATE: null
 
-TARSNAPPER_CONF: null
+# You'll need to upload these scripts by yourself
+TARSNAP_BACKUP_PRE_SCRIPT: null
+TARSNAP_BACKUP_POST_SCRIPT: null
+
```

Comment:
Actually I don't believe this is possible :(, or rather this solution is least painfull. The problem is that: if role A depends on tarsnap role we want to template configuration files in the context of role A (to have all variables present), only way (I found) to template configuration files in the context of  role A is to have that role upload all files.